mmv1/templates/terraform/examples/oracledatabase_autonomous_database_basic.tf.tmpl

```diff
@@ -0,0 +1,23 @@
+resource "google_oracle_database_autonomous_database" "{{$.PrimaryResourceId}}"{
+  display_name = "autonomousDatabase displayname"
+  autonomous_database_id = "my-adb-instance"
+  location = "us-east4"
+  project = "{{index $.Vars "project"}}"
+  database = "testADBdb"
+  admin_password = "123Abpassword"
+  network = data.google_compute_network.default.id
+  cidr = "10.5.0.0/24"
+  properties {
+    compute_count = "2"
```

Comment:
Can we omit any `properties` subfields that are not required here?

Reply:
The fields are optional but there are condition for example - 
1. user has to provide one of dataStorageSizeTb or dataStorageSizeGb.
2. user has to provide  computeCount or cpuCoreCount
Otherwise the prepare operation will itself fail

Reply:
We could consider using exactly_one_of for those pairs: https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#exactly_one_of

Non blocking, though.

Reply:
Will do as a follow up. Thanks for suggestions.